packages/notification/src/_styles.scss

```diff
@@ -201,6 +201,13 @@ $notification-slide-right: transform 300ms ease-out;
     margin-left: 6px;
   }
 
+  %ca-notification---toast & {
+    margin-left: 6px;
+    max-width: calc(100% - #{$spacing-xl});
+    overflow-wrap: break-word;
+    hyphens: auto;
```

Comment:
Maybe we can apply these styles based on a `@container` query? I'm not entirely sold on having a `isToast` prop for this 😅

Reply:
Hm Im getting a lint issue when trying to use it - unknown at rule container css(unknownAtRules)

Reply:
Probably worth digging a little into why, we'll need this for future work too. If it ends up being too deep, make a ticket and we can move on

Reply:
I added a container query - thoughts?

Reply:
That's like magic lol